Fix actions dropdown focus timing

PVince81 opened the pull request:
Because "@after-show" is still happening too early due to v-tooltip's
use of "requestAnimationFrame" we can't rely on its events to tell us
when the popover is actually visible.

This workaround introduces an additional Popover event "after-show" and
"after-hide" that are based on monitoring the inner "isOpen" property
which is the one toggling the CSS visibility.

The Actions popover logic now uses "after-show" to reliably apply the
focus on the first action element.

Fixes https://github.com/nextcloud/nextcloud-vue/issues/1760
See https://github.com/Akryum/v-tooltip/issues/661 for the upstream issue.

PVince81 commented on src/components/Popover/Popover.vue:
```diff
@@ -74,6 +75,24 @@ export default {
+				return this.$refs.popover.isOpen
+			},
+		  (val) => {
+			  if (val) {
```
weird indents... fixing...

PVince81 commented:
tldr; fixes keyboard navigation for after opening the menu :smile:

skjnldsv commented on src/components/Popover/Popover.vue:
```diff
@@ -74,6 +75,24 @@ export default {
+			},
+			(val) => {
+				if (val) {
+					this.$emit('after-show')
```
Please document those events

skjnldsv replied:
@PVince81 you can document them for our styleguide directly, no need for a readme :)
https://vue-styleguidist.github.io/docs/Documenting.html#events

PVince81 commented:
@skjnldsv docs added

PVince81 commented:
@skjnldsv nice, thanks for the hint. I've reverted the bits in the header and added docs close to `$emit`. Then squashed into the last commit.

Now it looks like this:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/277525/111340281-c852d000-8678-11eb-93d8-c1f37e14b12b.png">

skjnldsv reviewed (approved):
![giphy](https://user-images.githubusercontent.com/14975046/111343745-069dbe80-867c-11eb-98c7-c1ce0fb8af41.gif)
